### DISCUSSION TITLE
Local area requirements

### DISCUSSION THREAD

**Discussion by mattlucht** [digital-land/data-standards-backlog#52] [Planning consideration]
We've started exploring the local validation checklists to help us understand which planning considerations might add the most value if added to planning.data.gov.uk

**Comment by kieranwint** [comment]
After digging a little deeper into the numbers, it can be seen that the local validation checklists are highly variable in both the level of detail and the conveying of information.

Of the 335 LPAs, 271 have provided enough information to satisfy our chosen criteria of providing a hosting page, a document, and a start date. The 64 LPAs not currently included in the following statistics are omitted due to difficulty in satisfying the above criteria.

Of the 271 LPAs:
- 57 provided multiple variations of a checklist depending on application type
- 38.1% of local lists are dated within the last 2 years in line with regulation
- 38.9% are dated outside of the last 2 years (expired)
- Unknown Dates appeared 22.6%
- There was ONE instance of multiple dates linking with multiple document variations

- 23.7% in complete data format (YYYY/MM/DD)
- 49.6% in YYYY/MM format
- 3.7% providing the year only

In terms of the content of local checklists, although this hasn't been deeply analysed as of yet, there are some datasets that are beginning to appear more common that could be of value going forwards, for example, Air Quality Management Areas (AQMAs). More to follow.

**Comment by mattlucht** [comment]
We've been looking into local area requirements as we think that understanding more about the various planning requirements and what triggers them will help us when it comes to prioritising which datasets might offer the greatest value when added to planning.data.gov.uk.

We're fairly early on in our analysis, and have written a short blog post summarising where we've gotten to: https://digital-land.github.io/blog-post/digging-into-local-area-requirements/

**Comment by RichLimbrick** [comment]
This is great work as always - and really key to unlocking a more streamlined submission system - whether that's via the planning portal, PlanX, or some other platform. The Airtable you've produced is great and the triggers split by consideration, rule, and app type will really help us rationalise the triggers I think - they're currently too varied and sometimes not very objective.

**Reply by mattlucht** [reply]
thanks @RichLimbrick appreciate the feedback! We're going to add some more local area requirements from some additional LPAs with a view of seeing what "triggers" might bubble up as potential things of value to add to planning.data.gov.uk. Not solely how we'll assess priorities, but hopefully a useful indicator!

**Reply by RichLimbrick** [reply]
Sounds like a great idea. I think a venn diagram of triggers would show a lot of cross over which could allow us to minimise and standardise those (as far as possible). Good stuff.
